modules/fluid_properties/include/userobjects/SinglePhaseFluidProperties.h

```diff
@@ -405,3 +415,67 @@ class SinglePhaseFluidProperties : public FluidProperties
 };
 
 #pragma GCC diagnostic pop
+
+template <typename T>
+std::pair<T, T>
+SinglePhaseFluidProperties::makeZeroAndOne(const T & /*ex*/)
+{
+  return {T{0, 0}, T{1, 0}};
+}
+
+template <>
+inline std::pair<Real, Real>
```

Comment:
why the instantiation?

Reply:
Because you cannot construct a Real with the same constructor arguments as you can dual numbers